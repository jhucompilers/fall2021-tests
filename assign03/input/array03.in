PROGRAM arrao03;

    TYPE Empty = ARRAY 0 OF INTEGER;
    VAR a, b: Empty;

BEGIN
    -- a := b;
   -- this assignment will be invalid at run-time (due to out
   -- out of bounds array indices), but is allowed at compile
   -- time
   a[0] := b[0];
END.
