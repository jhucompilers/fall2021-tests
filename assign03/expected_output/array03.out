0,TYPE,Empty,ARRAY 0 OF INTEGER
0,VAR,a,ARRAY 0 OF INTEGER
0,VAR,b,ARRAY 0 OF INTEGER
